schema/core.md

```diff
@@ -158,11 +158,13 @@ creators is preserved.
  }
 ```
 
-### Date Created
+### Date Created, Date Modifie, and Date Published
 
-**DateCreated** is a property of `CreativeWork` that can be expressed using
-either the `Date` or `DateTime` classes. The **dateCreated** represents the
-date at which the dataset was initially generated. The `Date` class expects a
+**dateCreated** are properties of `CreativeWork` that can be expressed using
```

Comment:
Should this be "**dateCreated**, **dateModified**, and **datePublished** are properties of..."

Reply:
We should also move the dateModified and datePublished examples into this section of the text. Perhaps rename the subsection to "Dates" and we can describe it all in one place.

Reply:
I like the idea of calling the subsection "Dates". Commit 6ac5982 addresses these comments.